src/ctapipe/io/hdf5monitoringsource.py

```diff
@@ -284,14 +279,13 @@ def _process_pixel_statistics(self, file):
                     continue
                 # Instantiate the appropriate interpolator based on the table name
                 if "pedestal_image" in group._v_name:
```

Comment:
if you assign `name = group._v_name` the weird ruff line break further below should go away and the code should be a bit easier on the eyes.